### ISSUE TITLE
First results (locally)

### ISSUE THREAD

**Issue by plrepousername** [plehmann93/bdapro_winagg#4]
Hey Jeyhun,

first I have to say that I am sorry that I could not manage to make more progress till today, but I am very sick for 1 week now and therefore limited in the workload I can handle. I tried my best to do at least some reasonable progress.

Here are some first basic results I got when I run the experiments on my machine in a VM.
As I want to see the influences of the parameters Windowsize, Slidingsize , and Workload, I ran some experiments where two of these parameters were fixed. Therefore one can see how the latency behaves regarding the third parameter.

For the Windowsize:
![slidingsize1000workload100000](https://cloud.githubusercontent.com/assets/22885891/21555040/caeca456-ce14-11e6-9d7b-67061d22d45f.png)

For the Slidingsize:
![windowsize30000workload100000](https://cloud.githubusercontent.com/assets/22885891/21555043/d33b4252-ce14-11e6-9296-1710e8aa0fac.png)

For the workload:
![windowsize3000slidingsize2000](https://cloud.githubusercontent.com/assets/22885891/21555046/da61b692-ce14-11e6-9978-656ffeabaa96.png)

Here is one result of the yahoo benchmark:
![yahoo_benchmark](https://cloud.githubusercontent.com/assets/22885891/21555276/dbd9b306-ce16-11e6-8675-e9ba97d561aa.png)

In generell I would say that these results are like I would expect them if I take the results of the yahoo benchmark in account. 
Basically what these results here say is:
- For sliding windows an increasing windowsize does not affect the latency. 
- An increasing slidingsize increases the workload to compute for each window as well
- With a higher workload spark´s latency increases approximately linear
- Flink is not much effected on an increasing workload (maybe because of backpressure mechanism)

Of course these results have still a high deviation. I might want to run some experiments multiple time.
I also want to try different configurations of the parameters.
And of course I should try to work on the cluster.

**Comment by plrepousername** [comment]
@jeyhunkarimov here are some first results I made

**Comment by jeyhunkarimov** [comment]
@plehmann93 thanks for sharing. In general the results make sense although I have some questions. For now, I suggest you to start working on cluster and just apply the same experiments on cluster environment. I would like to discuss some points related with results in f2f meeting (on Monday?).  
I wish you and your family happy festive days and new year 2017.

**Comment by plrepousername** [comment]
@jeyhunkarimov Thanks. I wish you and your family a happy new year as well.
I went to the doctor today and he gave me a sick note until wednesday next week. Therefore it really depends how I feel. I could write you if I am feeling better so we can meet.

**Comment by plrepousername** [comment]
@jeyhunkarimov  If it is suitable for you we could meet tomorrow (tuesday) . At 11.00 ?

**Comment by jeyhunkarimov** [comment]
@plehmann93 ok
